Ambiguous-Squids/Bangazon_initial_site
# initial model setup pulled from first sprint

# Description
Setup of the models, in Django, that we will need for this project. Pulled over from first sprint.

# Related Ticket(s)
#14 

## Steps to Test
1. run migrations from project directory `python manage.py migrate`
2. Open the DB in DB Browser to verify contents
